lib/Plerd.pm

```diff
@@ -40,13 +40,26 @@ has 'database_path' => (
     isa => 'Str',
 );
 
+has 'tags_publication_path' => (
```

Comment:
All these public attributes ought to be documented in the appropriate section of the POD at the end of this file.